src/test/obj_ctl_arenas/obj_ctl_arenas.c

```diff
@@ -242,6 +258,64 @@ main(int argc, char *argv[])
 		ret = pmemobj_ctl_get(pop, "heap.narenas.total", &narenas_a);
 		UT_ASSERTeq(ret, 0);
 		UT_ASSERTeq(narenas_b + narenas_n, narenas_a);
+	} else if (t == 'f') {
+		int ret;
+		ret = pmemobj_ctl_set(pop, "heap.alloc_class.128.desc",
+				&alloc_class[0]);
+		UT_ASSERTeq(ret, 0);
+		ret = pmemobj_ctl_set(pop, "heap.alloc_class.129.desc",
+				&alloc_class[1]);
+		UT_ASSERTeq(ret, 0);
+
+		unsigned total;
+		unsigned arena_id;
+		ret = pmemobj_ctl_get(pop, "heap.narenas.total", &total);
+		UT_ASSERTeq(ret, 0);
+
+		for (unsigned i = total; i < 4; i++) {
```

Comment:
this will never be called on machines with more than 4 cores. Do something like this:
```
int arenas[NARENAS];
for (int i = 0; i < NARENAS; ++i) {
  pmemobj_ctl_exec(pop, "heap.arena.create", &arenas[i]);
}
```
and then whenever you allocate: `pmemobj_xalloc(..., POBJ_ARENA_ID(arenas[2]), ...);`